lighthouse-core/test/gather/computed/metrics/estimated-input-latency-test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Runner = require('../../../../runner');
+const EstimatedInputLatency = require('../../../../gather/computed/metrics/estimated-input-latency'); // eslint-disable-line
+const assert = require('assert');
+
+const trace = require('../../../fixtures/traces/progressive-app-m60.json');
+const devtoolsLog = require('../../../fixtures/traces/progressive-app-m60.devtools.log.json');
+
+/* eslint-env mocha */
+
+describe('Metrics: EIL', () => {
+  it('should compute a simulated value', async () => {
+    const artifacts = Runner.instantiateComputedArtifacts();
+    const settings = {throttlingMethod: 'simulate'};
+    const result = await artifacts.requestEstimatedInputLatency({trace, devtoolsLog, settings});
+
+    assert.equal(Math.round(result.timing), 104);
```

Comment:
this is a legitimate difference, basically the original trace has ~400ms of mainthread work that escapes in the freebie 500ms of a 5s window, in Lantern this gets 4x'd and isn't hidden anymore